### PR TITLE
Refactor/nw for loops

### PR DESCRIPTION
**Pull request by nware1066**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features

### Detailed Description / What does this PR do?
forLoops have been refactored to forEach prototypes, white space and comments have been removed userRepository
### Files modified:
userRepository.js, Users.js, DomUpdates.js
### Has This Been Tested?
- [ ] No
- [x] Yes

### Are there specific methods for testing?
tested by calling the data in the dom
### What are the relevant tickets?
#36 
### Checklist:
- [ ] My code has no unused/commented out code
- [ ] My code has no console logs
- [ ] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Activity.js [hunk 1]
```diff
@@ -1,39 +1,35 @@
...
+    return Math.round(this.numSteps * walkingUser.strideLength / 5280).toFixed(1);
+  }
+
+  compareStepGoal(userRepository) {
+    let userStepGoal = userRepository.users.find(user => {
+      return user.id === this.userId;
+    }).dailyStepGoal;
+    this.reachedStepGoal = this.numSteps >= userStepGoal;
   }
 }
```
Comment (Abdeboskey) [inline src/Activity.js:35]: Way cleaner without the comments, and with some added newLines. Now my eyes don't bleed when I look at it, thank you!

File: src/DomUpdates.js [hunk 2]
```diff
@@ -70,11 +70,11 @@ class DomUpdates {
...
     let dailyOz = document.querySelectorAll(".daily-oz");
-    for (var i = 0; i < dailyOz.length; i++) { // convert into forEach  <--------------- Hey!
-      dailyOz[i].innerText = user.addDailyOunces(
+    dailyOz.forEach((entry, i) => {
+      entry.innerText = user.addDailyOunces(
         Object.keys(sortedHydrationDataByDate[i])[0]
       );
-    }
+    });
   }
```
Comment (Abdeboskey) [inline src/DomUpdates.js:79]: Nice! way to use that index argument of the forEach

File: src/User.js [hunk 3]
```diff
@@ -138,37 +138,37 @@
...
+
   findTrendingStepDays() {
     let positiveDays = [];
-    for (var i = 0; i < this.activityRecord.length; i++) {
-      if (this.activityRecord[i + 1] && this.activityRecord[i].numSteps > this.activityRecord[i + 1].numSteps) {
-        positiveDays.unshift(this.activityRecord[i].date);
+    this.activityRecord.forEach((activity, i) => {
+      if (this.activityRecord[i + 1] && activity.numSteps > this.activityRecord[i + 1].numSteps) {
+        positiveDays.unshift(activity.date);
       } else if (positiveDays.length > 2) {
```
Comment (Abdeboskey) [inline src/User.js:147]: Yepppp

File: src/User.js [hunk 4]
```diff
@@ -138,37 +138,37 @@
...
+
   findTrendingStairsDays() {
     let positiveDays = [];
-    for (var i = 0; i < this.activityRecord.length; i++) {
-      if (this.activityRecord[i + 1] && this.activityRecord[i].flightsOfStairs > this.activityRecord[i + 1].flightsOfStairs) {
-        positiveDays.unshift(this.activityRecord[i].date);
+    this.activityRecord.forEach((activity, i) => {
+      if (this.activityRecord[i + 1] && activity.flightsOfStairs > this.activityRecord[i + 1].flightsOfStairs) {
+        positiveDays.unshift(activity.date);
       } else if (positiveDays.length > 2) {
```
Comment (Abdeboskey) [inline src/User.js:159]: Alright!

File: src/Activity.js [hunk 5]
```diff
@@ -1,39 +1,35 @@
...
-  constructor(data, userRepository) { // instantiated with data from dataset, and userRepository to access a specific user?
-    this.userId = data.userID; // stores unique user ID to link with User who created this activity
-    this.date = data.date; // stores the date that this activity data was logged
-    this.numSteps = data.numSteps; // stores the number of steps taken (per day?) ???
-    this.minutesActive = data.minutesActive; // stores the length of the activity logged in minutes
-    this.flightsOfStairs = data.flightsOfStairs; // stores flights of stairs walked (steps / 12?) ???
-    this.milesWalked = 0; // stores miles walked during activity
-    this.reachedStepGoal = null; // probably assigned a boolean if goal is reached in session
-    this.doActivity(userRepository); // calls this function on instantiation
+class Activity {
```
Comment (NicoleLatifi) [inline src/Activity.js:1]: Thank you for removing these comments.

File: src/DomUpdates.js [hunk 6]
```diff
@@ -70,11 +70,11 @@ class DomUpdates {
 
   updateHydrationCalendarCard(user, sortedHydrationDataByDate) {
     let dailyOz = document.querySelectorAll(".daily-oz");
-    for (var i = 0; i < dailyOz.length; i++) { // convert into forEach  <--------------- Hey!
-      dailyOz[i].innerText = user.addDailyOunces(
+    dailyOz.forEach((entry, i) => {
```
Comment (NicoleLatifi) [inline src/DomUpdates.js:73]: Great use of the forEach!

File: src/User.js [hunk 7]
```diff
@@ -138,37 +138,37 @@
...
     }, 0) / 7).toFixed(1);
   }
-  
+
   findTrendingStepDays() {
     let positiveDays = [];
-    for (var i = 0; i < this.activityRecord.length; i++) {
-      if (this.activityRecord[i + 1] && this.activityRecord[i].numSteps > this.activityRecord[i + 1].numSteps) {
-        positiveDays.unshift(this.activityRecord[i].date);
+    this.activityRecord.forEach((activity, i) => {
```
Comment (NicoleLatifi) [inline src/User.js:144]: Great use of forEach!

**Pull request merged**